src/backend/lib/utils.py

```diff
@@ -4,17 +4,29 @@
 from src.backend.lib.auth_utils import get_decoded_token
 from fastapi import HTTPException, Request
 
+from src.backend.lib.auth_decorator import authenticate_and_check_role
+
 # Setup logging configuration
 logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
 logger = logging.getLogger(__name__)
 
 
+async def auth_admin_dependency(request: Request):
+    return await authenticate_and_check_role(request, allowed_roles=["org:admin"])
+
+async def auth_admin_member_dependency(request: Request):
+    return await authenticate_and_check_role(request, allowed_roles=["org:admin", "org:member"])
+
+async def auth_admin_member_user_dependency(request: Request):
+    return await authenticate_and_check_role(request, allowed_roles=["org:admin", "org:member", "org:user"])
+
 class CustomerService:
     def __init__(self):
         self.db_manager = DatabaseManager()
 
     def get_customer_guid_from_token(self, request: Request):
         decoded_token = get_decoded_token(request)
+        logger.info(f"Decoded token: {decoded_token}")
```

Comment:
Create a new PR and Remove this line
logger.info(f"Decoded token: {decoded_token}")